### ISSUE TITLE
Hassio 0.98 

### ISSUE THREAD

**Issue by ninjax95** [biker91620/ha-cozytouch#10]
Hi,

I have Hassio 0.98.3, is it planned to have your addon run with this OS ?

Thanks

**Comment by thundergreen** [comment]
Yes you have to manually login into the homeassistant docker container (via portainer) and install the cozypy with this command:

```pip install cozypy```

Once you upgrade homeassistant you need to re-install the python package manually again
